cmake/re-config.cmake

```diff
@@ -5,8 +5,10 @@ find_package(Backtrace)
 find_package(Threads REQUIRED)
 find_package(ZLIB)
 find_package(OpenSSL)
+find_package(MbedTLS)
 
 option(USE_OPENSSL "Enable OpenSSL" ${OPENSSL_FOUND})
+option(USE_MBEDTLS "Enable MbedTLS" ${MBEDTLS_LIBRARY_FOUND})
```

Comment:
Looks like `MBEDTLS_LIBRARY_FOUND` does not work/exists, is it `MBEDTLS_FOUND`?

Reply:
I used common MbedTLS CMake finder that used in many projects, actually MbedTLS consists from 3 separate libraries and it's suffucient to check only main `MBEDTLS_LIBRARY` is found or not
https://github.com/widgetii/re/blob/mbedtls/cmake/FindMbedTLS.cmake#L3

Reply:
As far as I know `_FOUND` var are set by `find_package_handle_standard_args(MbedTLS DEFAULT_MSG`
not by `find_library`, so it has to be `MBEDTLS_FOUND`:

https://cmake.org/cmake/help/latest/module/FindPackageHandleStandardArgs.html:

```
The <PackageName>_FOUND variable will be set to TRUE if all the variables <required-var>... are valid and any optional constraints are satisfied, and FALSE otherwise.
```

`MBEDTLS_LIBRARY_FOUND` does not exists.

Reply:
You're right, I missed this part. Tested behavior both with library exist and absent conditions, and it works well